src/main/java/org/reduct/model/entry/Entry.java

```diff
@@ -0,0 +1,26 @@
+package org.reduct.model.entry;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import org.apache.commons.lang3.SerializationUtils;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@Builder
+public class Entry<BODY_CLASS extends Serializable> {
+    private String name;
+    private Long timestamp;
+    private final BODY_CLASS body;
```

Comment:
There is some meta information. Content type and labels.

Reply:
Not yet, not in this branch